CHANGELOG.md

```diff
@@ -3,6 +3,51 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.0-alpha.37
+
+_July 12, 2021_
+
+Big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🔎 Add the \"is empty\" and \"is not empty\" filter operators to date and number columns (#2274) @flaviendelangle
+- ⚡️ Only support @material-ui/core 4.12.0 or higher (#2281) @DanailH
+- 🐞 Fix a regression to not require @material-ui/x-license when using the DataGrid (#2295) @oliviertassinari
+- 👁️ Add `onViewportRowsChange` prop (#2038) @DanailH
+- 📃 Translate booleans when exporting rows to CSV (#2296) @m4theushw
+
+### `@material-ui/data-grid@v4.0.0-alpha.37` / `@material-ui/x-grid@v4.0.0-alpha.37`
+
+#### Changes
+
+- [DataGrid] Add @material-ui/styles as peer dependency (#2288) @m4theushw
+- [DataGrid] Add Sudanese Arabic (arSD) locale (#2269) @YassinHussein
```

Comment:
Maybe we should move this to the headlines?

Reply:
31.9 million native speakers, 7 billion people, 0.5% of the population